### PR TITLE
Add TLS verification tests

### PR DESCRIPTION
**Pull request by stormi**

### CODE REVIEW

File: lib/common.py [hunk 1]
```diff
@@ -152,4 +152,4 @@ def _param_get(host, xe_prefix, uuid, param_name, key=None, accept_unknown_key=F
             value = None
         else:
             raise
-    return value
```
Comment (benjamreis) [inline lib/common.py]: Is this related to this PR?

File: jobs.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
             "tests/misc",
             "tests/migration",
             "tests/snapshot",
+            "tests/xapi",
             "tests/xapi-plugins",
```
Comment (benjamreis) [inline jobs.py:28]: Is it related to this PR?

Comment (stormi) [reply jobs.py:28]: Yes. Every new test must belong to a job if we want them to be tested regularly.

**Review by gduperrey — APPROVED** [review]

**Review by Wescoeur — APPROVED** [review]
